frontend/models/product-list/types.ts

```diff
@@ -18,6 +18,12 @@ export interface ProductSearchHit {
    [attribute: string]: unknown;
 }
 
+export type PriceTier = {
+   default_variant_price: string | number;
+   min: string | number;
+   max: string | number;
```

Comment:
I think these are always supposed to be floats in algolia, so I don't think you need `string` here. Have you seen strings in practice? (Maybe floats are serialized as strings from Algolia?)

Reply:
No I did this just to be safe.. I didn't want to fail just because the format is a string